ext/shmop/tests/002.phpt

```diff
@@ -11,57 +11,81 @@ edgarsandi - <edgar.r.sandi@gmail.com>
 --FILE--
 <?php
 
-echo PHP_EOL, '## shmop_open function tests ##';
-    // warning outputs: invalid flag when the flags length != 1
-    var_dump(shmop_open(1338, '', 0644, 1024));
+echo PHP_EOL, '## shmop_open function tests ##', PHP_EOL;
 
-    // warning outputs: invalid access mode
-    var_dump(shmop_open(1338, 'b', 0644, 1024));
+    // Invalid flag when the flags length != 1
+    try {
+        shmop_open(1338, '', 0644, 1024);
+    } catch (ValueError $exception) {
+        echo $exception->getMessage() . "\n";
+    }
 
-    // warning outputs: unable to attach or create shared memory segment
+    try {
+        shmop_open(1338, 'b', 0644, 1024);
+    } catch (ValueError $exception) {
+        echo $exception->getMessage() . "\n";
+    }
+
+    // Warning outputs: Unable to attach or create shared memory segment
     var_dump(shmop_open(null, 'a', 0644, 1024));
 
-    // warning outputs: Shared memory segment size must be greater than zero
-    var_dump(shmop_open(1338, "c", 0666, 0));
+    // Shared memory segment size must be greater than zero
+    try {
+        shmop_open(null, 'a', 0644, 1024);
+    } catch (ValueError $exception) {
+        echo $exception->getMessage() . "\n";
+    }
+
+    //Shared memory segment size must be greater than zero
+    try {
+        shmop_open(1338, "c", 0666, 0);
+    } catch (ValueError $exception) {
+        echo $exception->getMessage() . "\n";
+    }
 
-echo PHP_EOL, '## shmop_read function tests ##';
-    // warning outputs: start is out of range
+echo PHP_EOL, '## shmop_read function tests ##', PHP_EOL;
+    // Start is out of range
     $shm_id = shmop_open(1338, 'n', 0600, 1024);
-    var_dump(shmop_read($shm_id, -10, 0));
+    try {
+        shmop_read($shm_id, -10, 0);
+    } catch (ValueError $exception) {
+        echo $exception->getMessage() . "\n";
+    }
     shmop_delete($shm_id);
 
-    // warning outputs: count is out of range
+    // Count is out of range
     $shm_id = shmop_open(1339, 'n', 0600, 1024);
-    var_dump(shmop_read($shm_id, 0, -10));
+    try {
+        shmop_read($shm_id, 0, -10);
+    } catch (ValueError $exception) {
+        echo $exception->getMessage() . "\n";
+    }
     shmop_delete($shm_id);
 
-echo PHP_EOL, '## shmop_write function tests ##';
-    // warning outputs: offset out of range
+echo PHP_EOL, '## shmop_write function tests ##', PHP_EOL;
+    // Offset out of range
     $shm_id = shmop_open(1340, 'n', 0600, 1024);
-    var_dump(shmop_write($shm_id, 'text to try write', -10));
+    try {
+        shmop_write($shm_id, 'text to try write', -10);
+    } catch (ValueError $exception) {
+        echo $exception->getMessage() . "\n";
+    }
     shmop_delete($shm_id);
 ?>
```

Comment:
As it completely changes anyway, please also unindent the whole test.

Reply:
Done!